### ISSUE TITLE
Unique field validation stops submission edits

### ISSUE THREAD

**Issue by mEsUsah** [verbb/formie#1211]
### Describe the bug

Validation of unique value does not check if the value is only on the current submission.

Editing a field in a submission that has the Unique Value light switch turned on will prevent the changes from being saved.

### Steps to reproduce

1. create a submission with a unique field 
2. edit submission with a unique field

### Form settings

- Multi-page form:  No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

3.7.61

### Plugin version

1.6.19

### Multi-site?

No

**Comment by engram-design** [comment]
Back-tracked from Formie 2 for Craft 4. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.20",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.21](https://github.com/verbb/formie/releases/tag/1.6.21)
